nlu.py

```diff
@@ -20,6 +20,34 @@ def _get_commands(text_sequence: [str]) -> set:
     return possible_commands
 
 
+def _consine_similarity(question: [str], text_sequence: [str]) -> float:
+    words = set(question+text_sequence)
+    question_frequency = _get_frequency(words, question)
+    text_frequency = _get_frequency(words, text_sequence)
+    return _cosine_theta(question_frequency, text_frequency)
+
+
+def _cosine_theta(a: [int], b: [int]):
+    denominator = (sum(pow(n, 2) for n in a) ** 0.5)*(sum(pow(n, 2) for n in b) ** 0.5)
```

Comment:
How about
```py
dot = lambda v: sum(pow(n, 2) for n in v) ** 0.5
dot(a) * dot(b)
```

Reply:
PEP8 not suggestion use lambda in the function(def), I will drag it out become a function.

Reply:
The point is they are the same expression, don't repeat yourself

Reply:
of course and it already fixed.